index.bs

```diff
@@ -320,42 +320,20 @@ TLS 1.3 {#tls-13}
 -----------------
 
 When an [=OSP Agent=] makes a QUIC connection to another agent, it must use
-[[!RFC8446|TLS 1.3]] to secure the connection.  TLS 1.3 must be used in the following
-application profile:
-
-* The [[!RFC7301|ALPN]] used must be "osp".
-* Only constant-time AEAD ciphers are allowed.
-    * OSP agents must support [[!RFC8446|TLS_AES_128_GCM_SHA256]].
-    * OSP agents should support [[!RFC8466|TLS_CHACHA20_POLY1305_SHA256]] and
-        [[!RFC8466|TLS_AES_256_GCM_SHA384]].
-* OSP agents must support the [[!RFC8446|ecdsa_secp256r1_sha256]] digital signature algorithm.
-    OSP agents should also support [[!RFC8466|ecdsa_secp384r1_sha384]] and
-    [[!RFC8466|ecdsa_secp521r1_sha512]]. OSP agents must not support additional digital
-    signature algorithms.
-    * `ecdsa_secp256r1_sha256` must be included in the `signature_algorithms`
-        extension.  `ecdsa_secp384r1_sha384` and `ecdsa_secp512r1_sha512` must be
-        included if they are supported by the agent.
-    * `secp256r1` must be included in the `supported_groups` extension.
-        `secp384r1` and `secp512r1` must be included if they are supported by the
-        agent.
-    * The `key_share` extension must be set to the cryptographic parameters for
-        the negotiated signature algoirithm.
+[[!RFC8446|TLS 1.3]] to secure the connection.  TLS 1.3 should be used with the
+following application-specific parameters to indicate that the connection will
+be used to communicate with a specific OSP Agent using OSP.  An OSP Agent may
+refuse incoming connections that lack these parameters.
+
+* The [[!RFC7301|ALPN]] used should be "osp".
```

Comment:
Why are the ALPN and SNI SHOULDs? It sounds like they should be MUSTs to ensure interoperability?

Reply:
That's a good point and makes the text more consistent.  Changed to MUST.